package.json

```diff
@@ -25,63 +25,64 @@
     "jestClear": "jest --clearCache"
   },
   "dependencies": {
-    "@patternfly/react-core": "^5.3.3",
+    "@patternfly/react-core": "^5.3.4",
     "@patternfly/react-icons": "^5.2.1",
-    "@patternfly/react-table": "^5.3.3",
-    "@redhat-cloud-services/frontend-components": "^4.2.11",
+    "@patternfly/react-table": "^5.3.4",
+    "@redhat-cloud-services/frontend-components": "^4.2.13",
     "@redhat-cloud-services/frontend-components-notifications": "^4.1.0",
-    "@redhat-cloud-services/frontend-components-utilities": "^4.0.13",
-    "@unleash/proxy-client-react": "^4.2.2",
+    "@redhat-cloud-services/frontend-components-utilities": "^4.0.15",
+    "@redhat-cloud-services/frontend-components-config": "^6.2.2",
```

Comment:
do you want to bump this to 6.2.3 (or newer)?  from a recent email we need to update this soon anyways 

```
By October 15, the webpack build configuration packages have to be updated to the latest version. The packages in question are:

    @redhat-cloud-services/frontend-components-config@6.2.3

    @redhat-cloud-services/frontend-components-config-utilities@3.2.2
```